site/src/main/java/uk/nhs/digital/ps/beans/Publication.java

```diff
@@ -58,23 +58,27 @@ public HippoBean getSelfLinkBean() {
         return HippoBeanHelper.getTaxonomyList(getKeys());
     }
 
-    public Series getParentSeries() {
+    public HippoBean getParentDocument() {
         assertPropertyPermitted(PropertyKeys.PARENT_SERIES);
 
-        Series seriesBean = null;
+        HippoBean parentBean = null;
 
         HippoBean folder = getParentBean();
         while (!HippoBeanHelper.isRootFolder(folder)) {
-            Iterator<Series> iterator = folder.getChildBeans(Series.class).iterator();
+            List<HippoBean> parentBeans = new ArrayList<>();
+
+            parentBeans.addAll(folder.getChildBeans(Series.class));
```

Comment:
a comment like "this will find first appearance of 'Series' or `Archive` document..." would me much appreciated.